src/readers/FeigReader.java

```diff
@@ -102,16 +108,17 @@ private void openUSBPort() throws FedmException, FePortDriverException, FeReader
 		fedm.addEventListener(this, FeIscListener.SEND_STRING_EVENT);
 		fedm.addEventListener(this, FeIscListener.RECEIVE_STRING_EVENT);
 
-		// Read important reader properties and set reader type in reader object.
+		// Read important reader properties and set reader type in reader
+		// object.
 		fedm.readReaderInfo();
```

Comment:
I see not reader type setting here as the comment say? Is it a side effect of "readReaderInfo"?